test/extended/cloud_controller_manager/ccm.go

```diff
@@ -52,17 +53,22 @@ var _ = g.Describe("[sig-cloud-provider][Feature:OpenShiftCloudControllerManager
 			g.Fail(fmt.Sprintf("Expected cloud-provider %v setting to indicate KCM relinquished cloud ownership", cloudProvider))
 		}
 
+		var systemdUnits []string
 		g.By("Getting masters MachineConfig")
-		masterkubelet, err := oc.AsAdmin().WithoutNamespace().Run("get").Args("machineconfig/01-master-kubelet", "-o=jsonpath={.spec.config.systemd.units[0].contents}").Output()
+		masterkubelets, err := oc.AsAdmin().WithoutNamespace().Run("get").Args("machineconfig/01-master-kubelet", "-o=jsonpath={.spec.config.systemd.units}").Output()
 		o.Expect(err).NotTo(o.HaveOccurred())
 		g.By("Expecting masters MachineConfig to contain cloud-provider as external for kubelet")
-		o.Expect(masterkubelet).To(o.ContainSubstring("cloud-provider=external"))
+		err = json.Unmarshal([]byte(masterkubelets), &systemdUnits)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		o.Expect(systemdUnits).To(o.ContainElement(o.SatisfyAll(o.ContainSubstring("name: kubelet.service"), o.ContainSubstring("cloud-provider=external"))))
```

Comment:
Are the systemd units in json, or yaml when they come out of the machineconfig?

Reply:
Systemd .ini format